app/ui/Box/SettingBox.tsx

```
@@ -0,0 +1,43 @@
+import Image from "next/image";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)